pkg/kubecost/summaryallocation.go

```diff
@@ -1170,6 +1170,101 @@ func (sas *SummaryAllocationSet) TotalCost() float64 {
 	return tc
 }
 
+// func to calculate average RAM efficiency over SummaryAllocationSet
+func (sas *SummaryAllocationSet) RAMEfficiency() float64 {
+	if sas == nil {
+		return 0.0
+	}
+
+	sas.RLock()
+	defer sas.RUnlock()
+
+	totalRAMBytesUsage := 0.0
+	totalRAMBytesRequest := 0.0
+	for _, sa := range sas.SummaryAllocations {
+		if sa.RAMBytesRequestAverage > 0 {
+			totalRAMBytesUsage += sa.RAMBytesUsageAverage
+			totalRAMBytesRequest += sa.RAMBytesRequestAverage
+		} else {
+			if sa.RAMBytesUsageAverage == 0.0 && sa.RAMCost == 0.0 {
+				totalRAMBytesUsage += 1.0
+				totalRAMBytesRequest += 1.0
+			}
```

Comment:
I'm not understanding this bit. Can you help with an explanation?

In general, my hope is that we can just add all the requests and usages together, simply; as in:
```go
for _, sa := range sas.SummaryAllocations {
    totalRAMBytesUsage += sa.RAMBytesUsageAverage
    totalRAMBytesRequest += sa.RAMBytesRequestAverage
}
```

Reply:
I wasnt sure if i need to maintain exact numbers or add 1.0 for usage and request.